pegasus/sites.v3/code.org/views/tenth_anniversary_tweets_testimonials.html

```diff
@@ -0,0 +1,7 @@
+<div class="tenth-anniversary-tweets-testimonials grid-container">
+  <blockquote class="twitter-tweet" data-dnt="true"><p lang="en" dir="ltr">As someone who started learning to code on my own, Hour of Code was the first resource I discovered and used. It made learning accessible, approachable, and fun and I’m so grateful that it sparked my journey into building software. <a href="https://t.co/aw0Cq2WdOX">https://t.co/aw0Cq2WdOX</a></p>&mdash; Isaac Fung (@ijfung) <a href="https://twitter.com/ijfung/status/1629972775229169664?ref_src=twsrc%5Etfw">February 26, 2023</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+  <blockquote class="twitter-tweet" data-conversation="none" data-dnt="true"><p lang="en" dir="ltr">Hadi &amp; everyone at <a href="https://t.co/QeeXGoNIoE">https://t.co/QeeXGoNIoE</a> -Thank you for bringing computer science to the forefront. I can&#39;t believe it has been 10 years since you started <a href="https://t.co/jI2HtRbaO4">https://t.co/jI2HtRbaO4</a> . Many of my students over the years have truly enjoyed learning how to code thanks to all of you!</p>&mdash; Stephanie Bermudez (@BellevilleHSLib) <a href="https://twitter.com/BellevilleHSLib/status/1630254432527974400?ref_src=twsrc%5Etfw">February 27, 2023</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+  <blockquote class="twitter-tweet" data-conversation="none" data-dnt="true"><p lang="en" dir="ltr">Still remember participating and completing the Hour of Code as a 13 year old kid back in 2013. I would definitely not have pushed to learn about computer science had I not done that. Very very grateful for this initiative ❤️</p>&mdash; Aditya Oberai (@adityaoberai1) <a href="https://twitter.com/adityaoberai1/status/1629930802653057024?ref_src=twsrc%5Etfw">February 26, 2023</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
Do these need to be translated?

Reply:
Tweet translation is handled on Twitter's end — it looks like they can be [automatically translated on Twitter](https://help.twitter.com/en/using-twitter/translate-tweets), but I don't think we have any control over whether these get translated on our site.

Reply:
Whoa fascinating. Do we need to hard-code the text in then? Our hard-coded text won't be translated, but it seems like Twitter does it for us, so will English users be seeing (or, potentially more likely, hearing via screenreader) the same text twice (once provided by Twitter, and once provided by us?)

I'm looking at this locally and see 
<img width="895" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/9142121/e28fc8d2-a4bf-45b4-8ddc-8c10ad6ea144">

Any idea why this wouldn't be loading locally?

Reply:
FWIW, I don't see any console errors

Reply:
Okay, I wish I understood this better, but I do see this pattern here https://developer.twitter.com/en/docs/twitter-for-websites/embedded-tweets/overview and it's hard to know exactly what's happening under the hood. Kelby, if you have tips for loading it locally, I'll take a closer look

Reply:
Okay, I loaded it locally (not sure what the problem was –– thanks, Kelby, for just encouraging me to try again!), and it looks great! I don't love relying on Twitter for the language piece, as reliability hasn't been their recent strong suit, but I think it's reasonable to rely on it for now.

<img width="903" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/9142121/312bb059-3db9-4a06-ad70-7c6dad205a9a">